spec/sample-data/valid/control-statement-03.json

```diff
@@ -0,0 +1,18 @@
+{
+  "subject_entity": {
+    "entity_properties": {
+      "company_number": "09876543",
+      "jurisdiction_code": "gb"
+    },
+    "entity_type": "company"
+  },
+  "no_owners_reasons": [
+    "PSC contacted but no response: The company has given a notice under section 790D of the Act which has not been complied with"
+  ],
+  "control_level": "unknown",
```

Comment:
Can you move `control_level` above `no_owners_reasons`? Makes it easier to compare with the others.